plugins/chat/config/locales/client.en.yml

```diff
@@ -687,3 +687,7 @@ en:
       sections:
         chat:
           title: Chat
+    sidebar:
+      panels:
+        chat:
+          label: Chat
```

Comment:
Is this being used already? If not I think we should leave it out from this commit.